core/arch/arm/kernel/secstor_ta.c

```diff
@@ -5,11 +5,11 @@
 
 #include <tee/tadb.h>
 #include <kernel/user_ta.h>
-#include <kernel/user_ta_store.h>
+#include <kernel/ts_store.h>
```

Comment:
move 1 line up. here and few other source files.

Reply:
Done